projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -30,7 +31,7 @@ enum SupportedContentTypes {
 @Component({
   selector: 'mv-media-viewer',
   templateUrl: './media-viewer.component.html',
-  styleUrls: ['styles/main.scss', './media-viewer.component.scss']
+  encapsulation: ViewEncapsulation.None
```

Comment:
same as above, do we need to specify this if there are no styles defined in the component anyway

Reply:
same as above